doc/source/Tutorials/BeginnerTutorial/index.md

```diff
@@ -283,7 +316,7 @@ Congratulations! You have successfully rendered your first image. Gaffy is curre
 
 ## Pinning an Editor to a Node ##
```

Comment:
Continuing my earlier comment about too much depth, this section could also use shortening at a later point. Again, it's useful, but it stops the tutorial flow in its tracks. A simple prescriptive instruction telling the user to do this would suffice.